streams/actions/src/main/java/com/hortonworks/streamline/streams/actions/topology/service/TopologyActionsService.java

```diff
@@ -355,4 +375,12 @@ public StreamCatalogService getCatalogService() {
         return catalogService;
     }
 
+    private Exception convertThrowableToException(Throwable e) {
+        if (e.getCause() instanceof Exception) {
+            return (Exception) e.getCause();
+        } else {
+            return new RuntimeException(e);
```

Comment:
1. Shouldn't it be `new RuntimeException(e.getCause())` ?
2. Does this handle if e is `IgnoreTransactionRollbackException` and e.getCause() is null?

Reply:
1. Yes I just fixed it.
2. No, but given that IgnoreTransactionRollbackException is just a marker exception it should provide valid exception. If we want to ensure this, null check should be landed to the constructor of IgnoreTransactionRollbackException.